packages/ui/src/Integration/IntegrationDetailHistoryListView.css

```diff
@@ -0,0 +1,12 @@
+.integration-detail-history-list-view {
+    background-color: white;
+}
+
+.integration-detail-history-list-view .list-description {
+    font-weight: bold;
+    padding-top: 20px;
+}
+
+.list-view-pf-view {
+    margin-top: 0;
```

Comment:
Better namespace this or it will clash with other list views in the app

Reply:
Good point

Reply:
Made some changes, I think this could use improvements, but I wonder if is ok for now?

Reply:
there is no need for the `c-` prefix actually, but yeah this is good

Reply:
Okay removed it!